.golangci.yaml

```diff
@@ -0,0 +1,19 @@
+run:
+  timeout: 1m
+  tests: true
+
+linters:
+  enable:
+    - asciicheck
+    - forcetypeassert
+    - gocritic
+    - godot
+    - gofmt
+    - goimports
+    - misspell
+    - stylecheck
+
+linters-settings: # please keep this alphabetized
```

Comment:
The comment makes sense, but then we should also have it above.

Reply:
Moved to top.